covalent/executor/base.py

```diff
@@ -147,6 +155,17 @@ def __init__(
         self.time_limit = time_limit
         self.retries = retries
 
+        if not workdir:
+            short_name = self.short_name()
+            try:
+                workdir = get_config(f"executors.{short_name}.workdir")
+            except KeyError:
+                workdir = os.path.join(os.environ["HOME"], "covalent", "workdir")
```

Comment:
Probably better if we use `.cache/covalent/workdir` here instead and make this variable a part of defaults in `covalent/_shared_files/defaults.py::get_default_dispatcher_config` so that we can directly get it from config (except for the case of `_EXECUTOR_PLUGIN_DEFAULTS` because `get_config` isn't called when setting those values)  instead of writing the entire path every time.

Reply:
Done!